src/modules/collections/dto/collections.dto.ts

```diff
@@ -12,6 +12,15 @@ export class CreateOrUpdateCollectionDto {
 		example: 'Favorites',
 	})
 	name: string;
+
+	@IsString()
+	@IsOptional()
+	@ApiPropertyOptional({
+		type: string,
+		description: 'The description of the collection',
```

Comment:
we're migrating all "collection" names to "folder" when you're touching the code or creating new code
```suggestion
		description: 'The description of the folder',
```